test/e2e_test/RefDecoder.cc

```diff
@@ -418,3 +418,13 @@ void RefDecoder::trans_video_frame(const void* image_handle,
     frame.timestamp =
         init_timestamp_ + ((uint64_t)dec_frame_cnt_ * frame_interval_);
 }
+
+void RefDecoder::control(int ctrl_id, int arg) {
```

Comment:
Do we really need this function? Its a simple wrapper of original control function, but only called once.